base/doc/ltnews41.tex

```diff
@@ -244,6 +244,14 @@ \subsection{Socket and plug conditionals}
 %
 \githubissue{1577}
 
+\subsection{Accessing the current counter}
+Counter commands such as \cs{alph}, \cs{stepcounter}, may now have
+argument \texttt{*} to denote the current counter (as used by
```

Comment:
have the argument

Reply:
I guess so, it's probably ok as it is as well, but ...